client/components/domains/register-domain-step/index.jsx

```diff
@@ -109,13 +110,15 @@ const RegisterDomainStep = React.createClass( {
 		basePath: React.PropTypes.string.isRequired,
 		suggestion: React.PropTypes.string,
 		domainsWithPlansOnly: React.PropTypes.bool,
-		isSignupStep: React.PropTypes.bool
+		isSignupStep: React.PropTypes.bool,
+		searchDelay: React.PropTypes.number.isRequired
 	},
 
 	getDefaultProps: function() {
 		return {
 			onDomainsAvailabilityChange: noop,
-			analyticsSection: 'domains'
+			analyticsSection: 'domains',
+			searchDelay: parseInt( abtest( 'domainSearchDelay' ) )
```

Comment:
If we're passing any props, let's just use the value from abtest directly to keep it simple.